MusicTaste/MusicTaste/Cells/MatchCell.m

```diff
@@ -0,0 +1,21 @@
+//
+//  MatchCell.m
+//  MusicTaste
+//
+//  Created by Aman Abraham on 7/6/22.
+//
+
+#import "MatchCell.h"
+
+@implementation MatchCell
+
+- (void)awakeFromNib {
+    [super awakeFromNib];
+}
+
+- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
+    [super setSelected:selected animated:animated];
+    
+}
```

Comment:
If a function is *only* calling its super, it can be removed. You may not know up front if you'll want to use them later, so it's ok to leave some of them. But ideally you're coming back through and removing some of these as you determine you don't need them.